### PR TITLE
Syntax highlighting in README.rst

### PR DESCRIPTION
**Pull request by batisteo**
It will be more readable on Github and hopefully on PyPI

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -47,8 +47,8 @@ basepython = python2.7
 deps =
     docutils
 commands =
-    {envbindir}/rst2html.py --report=info --halt=warning README.rst /dev/null
-    {envbindir}/rst2html.py --report=info --halt=warning CHANGES.rst /dev/null
+    {envbindir}/rst2html.py --syntax-highlight=short --report=info --halt=warning README.rst /dev/null
```
Comment (SmileyChris) [inline tox.ini]: According to travis, seems like this isn't valid in Python 2.7?

File: README.rst [hunk 2]
```diff
@@ -148,7 +156,9 @@ When you create your form, you can use this custom widget like normal::
             widgets = {'country': CountrySelectWidget()}
 
 Pass a ``layout`` text argument to the widget to change the positioning of the
-flag and widget. The default layout is::
+flag and widget. The default layout is:
+
+  .. code:: jinja
```
Comment (SmileyChris) [inline README.rst]: this is python, not template code

File: tox.ini (hunk 1 above)
Comment (batisteo) [reply tox.ini]: Yes, even without `--syntax-highlight=short` (see previous Travis build) 🙁

**Comment by batisteo** [comment]
It’s sad to take advantage of the `::` shortcut because of Python 2.
But that’s a detail anyway.

**Pull request merged**